pkgs/development/python-modules/wandb/default.nix

```diff
@@ -137,6 +143,8 @@ buildPythonPackage rec {
   preCheck = ''
     export HOME=$(mktemp -d)
     export BOKEH_CDN_VERSION=${bokeh.version}
+    # Prevent conflict with unittest conftest
```

Comment:
please avoid including comments in shell strings. they unnecessarily impact drv hashes

also do we have a github issue for this one? why is there a conflict? why can't we just disable that test?

Reply:
> please avoid including comments in shell strings. they unnecessarily impact drv hashes

sure.

> also do we have a github issue for this one? why is there a conflict? why can't we just disable that test?

It could be controlled by pytest flags.

Reply:
I set pytestFlagsArray to run only the tests under tests/pytest_tests.